src/overlays/actors/ovl_Bg_Gnd_Darkmeiro/z_bg_gnd_darkmeiro.h

```diff
@@ -10,10 +10,29 @@ typedef void (*BgGndDarkmeiroActionFunc)(struct BgGndDarkmeiro*, GlobalContext*)
 
 typedef struct BgGndDarkmeiro {
     /* 0x0000 */ DynaPolyActor dyna;
-    /* 0x0164 */ char unk_164[0x8];
+    /* 0x0164 */ u16 actionState;    /* Bitflags for whether a block or timer is on.
+                                        0x8 for timer 2, 0x4 for timer 1, 0x2 for blocks.   */
+    /* 0x0166 */ s16 timer1;         /* Timer for switch flag N+1. Blocks use it to count 
+                                        down to vanishing when their switch flag is cleared.*/
+    /* 0x0168 */ s16 timer2;         /* Timer for switch flag N+2. Blocks use it to control
+                                        their transparency (does not work).                 */
```

Comment:
in general i think itd be beneficial to keep these comments short and to the point. i personally aim to try to fit them on one line with a // comment (same for the enum below if possible)
not a hard guideline to go by, just a personal preference of mine and keeps documentation from getting too wordy